### PR TITLE
Update history_graph.markdown

### PR DESCRIPTION
**Pull request by Caswell1000**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/history_graph.markdown [hunk 1]
```diff
@@ -73,5 +73,18 @@ history_graph:
     # refresh: 0
 
 ```
+NOTE: In order to ensure that the graphs appear in the UI, you have to add the graphs that have been created in a new group in groups.yaml, the example below illustrates:
```
Comment (frenck) [inline source/_components/history_graph.markdown:76]: This statement is actually incorrect. It doesn't have to be in `groups.yaml`, secondly, that is depended on your setup.

File: source/_components/history_graph.markdown [hunk 2]
```diff
@@ -73,5 +73,18 @@ history_graph:
...
 
+```
+# this should be placed in groups.yaml
+
+graphs: # this is a name of your choice
+name: Graphs
```
Comment (frenck) [inline source/_components/history_graph.markdown:82]: Indenting totally got lost here...

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_components/history_graph.markdown (hunk 1 above)
Comment (Caswell1000) [reply source/_components/history_graph.markdown:76]: Hi,

I followed the instructions on the instructions, and the graph did not appear, it only appeared after I placed it in groups.yaml, perhaps the rephraing could be done thus: 

"Depending on the the setup, the graphs might not appear in the UI, to ensure they do appear or if they do not appear, place them in groups.yaml as follows"


```
# preferebly name of graph should be descriptive, entities should be in snake_case notation

Graphs:
name: graphs
view: yes
icon: mdi:chart-line-variant
entities:
  - history_graph.lights_graph
  - history_graph.temperature
```

Comment (frenck) [reply source/_components/history_graph.markdown:76]: That sounds like a solution, yes.

**Comment by frenck** [comment]
@Caswell1000 Are you planning on moving this PR forward?

**Comment by frenck** [comment]
PR running stale, closing it.
Feel free to re-open when ready.

**Pull request closed without merging**